### PR TITLE
Topics FPD module: initial release

### PR DESCRIPTION
**Pull request by dgirardi**
## Type of change

- [x] Feature

## Description of change

This adds a new `topicsFpdModule` that queries the [topics api](https://developer.chrome.com/docs/privacy-sandbox/topics/) to populate `user.data`.

Since `document.browsingTopics()` always returns an empty list for me, I need volunteers to test this in the real world.

## Other information

Closes https://github.com/prebid/Prebid.js/issues/7968
Depends on https://github.com/prebid/Prebid.js/pull/8626
Documentation TBD

### CODE REVIEW

File: modules/topicsFpdModule.js
Comment (vkimcm) [inline modules/topicsFpdModule.js:26]: testing on Chrome beta, when you get topics like this
`[{"configVersion":"chrome.1","modelVersion":"2206021246","taxonomyVersion":"1","topic":301,"version":"chrome.1:1:2206021246"},{"configVersion":"chrome.1","modelVersion":"2206021246","taxonomyVersion":"1","topic":3,"version":"chrome.1:1:2206021246"}]`
and datum looks like this
`[{"name":"domain","ext":{"segtax":600,"segclass":"1"},"segment":[{"id":"301"},{"id":"3"}]}]`

**Review by vkimcm — APPROVED** [review]

**Review by patmmccann — APPROVED** [review]

**Comment by dgirardi** [comment]
I reverted the merge of this and reopened https://github.com/prebid/Prebid.js/pull/8646 - to get a second review on making FPD modules asynchronous, and preferably wait on the merge of https://github.com/prebid/Prebid.js/pull/8626

File: modules/topicsFpdModule.js
Comment (vkimcm) [reply modules/topicsFpdModule.js:26]: yes, it was placeholder

Comment (patmmccann) [reply modules/topicsFpdModule.js:26]: segclass should be "2206021246" here @dgirardi , not 1

**Pull request merged**
